arevyn24-assignment-4/src/arevyn24/assignment/pkg4/Assignment4Methods.java

```diff
@@ -0,0 +1,329 @@
+    public static void gradeCalc(Scanner keyboard) {
+      //IN CLASS PROJECT
+        Double yourPoints;
+        Double maxPoints;
+        Double percentage;
+        char grade;
+        String score;
+
+         print("GRADE CALCULATOR\n");
+         print("\tEnter YOUR SCORE: ");
+         yourPoints = keyboard.nextDouble(); 
+         print("\tEnter MAX POINTS: ");
+         maxPoints = keyboard.nextDouble(); 
+         percentage = (yourPoints/maxPoints) * 100;
+
+         if(percentage >= 90) {
+                grade = 'A';
+         } else if (percentage >= 80) {
+             grade = 'B';
+         } else if (percentage >= 70) {
+             grade = 'C';
+         } else if (percentage >= 60) {
+             grade = 'D';
+         } else {
+             grade = 'F';
+         };
+         
+         score = String.format("Your grade: %.2f" + " " + grade + "\n\n", percentage);
+         print(score);
+        
+    };      //end gradeCalc method
+    
+    
+    /**
+     * loopingLyrics() accepts a number from the user that stores in the variable
+     * representing the number of times a loop executes. The loop prints array items
+     * one by one. User can repeat or return to menu.
+     * @param keyboard Scanner accepts user input
+     */
+    public static void loopingLyrics(Scanner keyboard) {
+      //LOOPING LYRICS
+        Integer numLoops;
+        /*Each line of the song is stored in the lyrics array. The user
+          chooses how many lines of lyrics will be printed (also the number
+          of times the loop will be executed).
+        */
+        String[] lyrics = {
+          "Adventure Time,", "Come on, grab your friends", "We'll go to" , 
+            "very distant lands", "With Jake the Dog", "And Finn the Human",
+            "The fun will never end,", "It's Adventure Time!"
+        };    
+
+        //title and description
+        print("SONG LYRICS LOOP\n" + "\tEnter a number from"
+                + " 1-8. This will be the number of times the loop"
+                + " executes: ");
+        numLoops = keyboard.nextInt();
+
+        do { 
+            /*If input is not a number between 1-8, prompt the user
+              to enter a new value. Keep looping until they enter a
+              correct value*/
+            while (numLoops > 8 || numLoops <= 0) {
+                print("Oops! Try again. Enter a number from"
+                                    + " 1-8.\n");
+                numLoops = keyboard.nextInt();
+            }
+            /*
+              In this for loop, the variable i represents the location
+              inside the lyrics[] array. numLoops is the user-chosen
+              number that decides how many lines of lyrics will be 
+              printed Each time the loop runs, i is increased, making
+              the next line of the song print out
+            */
+            for(int i=0; i < numLoops; i++) {
+                print("\t\t" + lyrics[i] +"\n");
+            } 
```

Comment:
Perfect. When we do arrays I will cover list comprehensions and we will learn a new kind of loop that is perfect for this case. :+1: